### ISSUE TITLE
Creating the "HOTKEYS" 

### ISSUE THREAD

**Issue by tvboxprivada** [libretro/RetroArch#16306]
Suggestion to the Retroarch team

Creating the "HOTKEYS" button option within the settings/input/retropadbinds path would be ideal by eliminating
several errors or incompatibility with the use of the select button

**Comment by RobLoach** [comment]
You are able to change the Hotkey Enable button. Could you expand on what you're asking?

**Comment by tvboxprivada** [comment]
Possuo diferentes tipos de gamepads, e para cada tipo de gamepad, é necessário uma configuração diferente. Isso inclui a configuração da tecla de atalho, mas infelizmente atualmente o "Retroarch" não disponibiliza a opção de Salvar qual o tipo de tecla, será usada individualmente para determinado tipo de gamepad.

O que ocasiona o não salvamento da opção da tecla de atalho com as demais teclas, todas as vezes que eu troco um gamepad, sou obrigado a reconfigurar todas as teclas que estão no menu "Hotkey".

Na imagem logo abaixo vemos a "Button 8" selecionado, se eu troco o meu "Gamepad", esse botão não acompanha as mesmas configurações do novo "Game.


![Tela Atalhos](https://github.com/libretro/RetroArch/assets/149258928/acac24ae-1fe4-4218-9dc9-746e88231bc6)

A minha sugestão é bem simples: alterar o momento em que o botão "Hotkey" deverá ser configurado, ou seja, a tecla "Hotkey" deverá ser configurada no menu "Port Controls", junto com as demais teclas, pois desta forma o botão que define a tecla "Hotkey" será salva, juntamente com as demais teclas. Eliminando a necessidade da reconfiguração desta tecla, todas as vezes que um "Gamepad" é substituido.

![Tela Config Keyboards](https://github.com/libretro/RetroArch/assets/149258928/0d0edd9d-af64-4d85-a29a-fc87b459e72d)

Com essa mudança a opção de definição da tecla "Hotkey", poderia ser eliminada, ou apenas "congelada", dentro do menu Hotkey. 

![Eleminar Hotkey](https://github.com/libretro/RetroArch/assets/149258928/45e6bf17-e64c-42f6-b09b-944dc5600e71)

Caso permaneça alguma dúvida no problema que relatei, ou a respeito da solução que sugeri, basta me questionar novamente.

**Comment by zoltanvb** [comment]
Even though this is currently not possible from menu, you can work around it by autoconfig file editing. After doing the assignment, the respective config row can be moved from retroarch.cfg and added to the autoconfig file that is being used, random example:
`input_toggle_slowmotion_btn = "h0left"`
It requires file editing, and may get overwritten if autoconfig profiles are refreshed.

**Comment by tvboxprivada** [comment]
Yes, today I have 6 retroarch.cfg files, for the 6 types of gamepads I have.
But this is almost idiotic, because the Retroarch team just needs to map the buttons in the configuration mode in the "slave" style.
Basically, when configuring the gamepad for the first time, it would be co-relating when I click on button 1=A, 2=B, 3=C and so on.
I even sent a message explaining in detail, even with an organization chart. But I was completely ignored in the post.
To this day I don't understand how this hasn't been implemented in Retroarch yet.
